packages/flutter_tools/test/general.shard/base/analyze_size_test.dart

```diff
@@ -0,0 +1,163 @@
+import 'package:file/memory.dart';
+import 'package:flutter_tools/src/base/analyze_size.dart';
+import 'package:flutter_tools/src/base/file_system.dart';
+import 'package:flutter_tools/src/base/logger.dart';
+import 'package:flutter_tools/src/base/process.dart';
+
+import '../../src/common.dart';
+import '../../src/context.dart';
+
+void main() {
+  MemoryFileSystem fileSystem;
+  BufferLogger logger;
+  FakeProcessManager processManager;
+
+  const FakeCommand unzipCommmand = FakeCommand(
```

Comment:
for the consts here, I would hoist them out of main at put them at the top level